app/views/foods/index.html.erb

```diff
@@ -0,0 +1,24 @@
+<h2>
+  Here's a list of your foods!
+</h2>
+<%= link_to(foods_path) do %>
+  Add Food
+<% end %>
+<ul>
+  <% @foods.each do |food| %>
+    <li>
+      <span>Food</span>
+      <span>Measurement unit</span>
+      <span>Unit price</span>
+      <span>Quantity</span>
+      <span>Actions</span>
+    </li>
```

Comment:
- [ ] [OPTIONAL]These are the table titles row and should be out of the cycle each

Reply:
Oh man hahaha I didn't see I put it inside the each method, thank!